src/Mailgun/Resource/Api/Suppressions/BaseResponse.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Suppressions;
+
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * Serves only as an abstract base for suppressions API code.
+ *
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+abstract class BaseResponse implements ApiResponse
+{
+    /**
+     * @var string
+     */
+    private $webhook;
+
+    /**
+     * @var string
+     */
+    private $message;
+
+    /**
+     * @param string $address
+     * @param string $message
+     */
+    public function __construct($address, $message)
+    {
+        $this->address = $address;
+        $this->message = $message;
+    }
+
+    /**
+     * @param array $data
+     *
+     * @return static
```

Comment:
I would say to change this to `@return BaseResponse` due we're returning a `BaseResponse` object.